### PR TITLE
transition to the PP 20170801

### LINKED ISSUE #696: There is a new dated version to the Patent Policy
Doh! We switched to https://www.w3.org/Consortium/Patent-Policy-20170801/  and didn't update our tooling...

Needs coordination with @tabatkins and @marcoscaceres

### PR DESCRIPTION
**Pull request by deniak**
Support for the new patent policy that was released last August.
The PR allows the use of the old patent policy until the end of this month. This should give enough time to update the spec generators.
Fix #696 

cc @plehegar, @marcoscaceres, @tabatkins

### CODE REVIEW

**Comment by marcoscaceres** [comment]
Ok, sent: 
https://github.com/w3c/respec/pull/1388 

Tagged @deniak and @plehegar for review.

**Comment by plehegar** [comment]
@wseltzer , what date would you like us to use for the text?

**Comment by plehegar** [comment]
besides the date to check, LGTM

**Comment by wseltzer** [comment]
@deniak and @plehegar it's still the same old patent policy, with some errata applied, so the date should be "5 February 2004 (updated 1 August 2017)"

**Comment by wseltzer** [comment]
I think we might be getting too fancy here. Why not use the undated PP link https://www.w3.org/Consortium/Patent-Policy/ for everything except the primary "This document was produced by a group operating under the 5 February 2004 (Updated 1 August 2017) W3C Patent Policy."

**Comment by plehegar** [comment]
if we're going to use the undated link version of the Patent Policy, can we also get rid of the date in the text and simply say:
This document was produced by a group operating under the <a href="https://www.w3.org/Consortium/Patent-Policy/">W3C Patent Policy</a>."
After all, we don't seem to make substantive changes to the policy (unlike the Process) so amybe we can simplify things further?

**Comment by deniak** [comment]
> Shouldn't we edit `lib/rules.json` too, to update the [boilerplate suggested for rule  `sotd/pp, patPolReq`](https://www.w3.org/pubrules/doc/rules/?profile=WD#patPolReq)?

@tripu indeed. I'll update the rule after we agree on the link and text to use.

File: test/docs/sotd/pp-20170801.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
     http://www.w3.org/TR/.</em></p>
     <p>
       This document was produced by a group operating under the
-      <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/">5 February 2004 (updated 1 August 2017)
+      <a href="https://www.w3.org/Consortium/Patent-Policy-20170801/">
```
Comment (plehegar) [inline test/docs/sotd/pp-20170801.html]: should be https://www.w3.org/Consortium/Patent-Policy/

File: lib/rules.json [hunk 2]
```diff
@@ -1324,7 +1324,7 @@
...
...ource\"> <span style=\"font-style: italic\">Include this source code</span>:<br><code>&lt;p&gt; This document was produced by a group operating under the &lt;a href=\"https://www.w3.org/Consortium/Patent-Policy-20170801/\"&gt;W3C Patent Policy&lt;/a&gt;. W3C maintains a &lt;a rel=\"disclosure\" href=\"@@URI to IPP status or other page@@\"&gt;public list of any patent disclosures&lt;/a&gt; made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains &lt;a href=\"https://www.w3.org/Consortium/Patent-Policy-20170801/#def-essential\"&gt;Essential Claim(s)&lt;/a&gt; must disclose the information in accordance with &lt;a href=\"https://www.w3.org/Consortium/Patent-Policy-20170801/#sec-Disclosure\"&gt;section 6 of the W3C Patent Policy&lt;/a&gt;. &lt;/p&gt;</code></div> <p>Note: Contact the Communications Team for suitable adaptations in the case where a document has been published jointly by more than one group. In the adaptation, be sure that the text for informative-only specs or specs not going to Rec is the same as the standard text. </p>",
```
Comment (plehegar) [inline lib/rules.json]: use https://www.w3.org/Consortium/Patent-Policy/

**Comment by deniak** [comment]
@wseltzer, @plehegar, I commited the updated rule. Here's the markup we are looking for now:
```html
<p>
This document was produced by a group operating under the <a href="https://www.w3.org/Consortium/Patent-Policy/"> <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a>. <abbr title="World Wide Web Consortium">W3C</abbr> maintains a <a href="https://www.w3.org/2004/01/pp-impl/40318/status" rel="disclosure">public list of
any patent disclosures</a> made in connection with the deliverables of the group; that page also includes instructions for disclosing a patent. An individual who has actual knowledge of a patent which the individual believes contains <a href="https://www.w3.org/Consortium/Patent-Policy/#def-essential">Essential Claim(s)</a> must disclose the information in accordance with <a href="https://www.w3.org/Consortium/Patent-Policy/#sec-Disclosure">section 6 of the <abbr title="World Wide Web Consortium">W3C</abbr> Patent Policy</a>.
</p>
```

Please confirm the markup is ok so we can deploy ASAP.

@tripu during a call today, we agreed to extend the transition period until end of Nov. I also addressed your comment about updating the rule description.

**Pull request merged**